schemas/v1/14_vm_messages_migration.go

```diff
@@ -0,0 +1,21 @@
+package v1
+
+func init() {
+	patches.Register(
+		14,
+		`
+{{- if and .SchemaName (ne .SchemaName "public") }}
```

Comment:
I am surprised this is required.

Reply:
ahh right, since we are using the `create_hypertable` method I think this is required.